astropy/io/misc/asdf/types.py

```diff
@@ -21,12 +22,27 @@ def __new__(mcls, name, bases, attrs):
         cls = super(AstropyTypeMeta, mcls).__new__(mcls, name, bases, attrs)
         # Classes using this metaclass are automatically added to the list of
         # astropy extensions
-        if cls.organization == 'stsci.edu' and cls.standard == 'asdf':
+        if cls.organization == 'astropy.org' and cls.standard == 'astropy':
+            _astropy_types.add(cls)
+        elif cls.organization == 'stsci.edu' and cls.standard == 'asdf':
             _astropy_asdf_types.add(cls)
 
         return cls
 
 
+@six.add_metaclass(AstropyTypeMeta)
+class AstropyType(CustomType):
+    """
+    This class represents types that have schemas that are defined by Astropy.
+    standard, but have tags that are implemented within astropy.
```

Comment:
Some copy & paste error? Here or in the `IMPORTANT` below.

Reply:
Good catch!